frontend/packages/knative-plugin/src/actions/providers.ts

```diff
@@ -107,7 +102,7 @@ export const useKnativeServiceActionsProvider = (resource: K8sResourceKind) => {
         ? [DeleteResourceAction(kindObj, resource)]
         : [CommonActionFactory.Delete(kindObj, resource)]),
       ...(resource?.metadata?.labels?.['function.knative.dev'] === 'true' &&
-      localStorage.getItem(KNATIVE_SERVERLESS_FUNCTION_TEST) === 'true'
+      resource?.metadata?.labels?.[PRIVATE_KNATIVE_SERVING_LABEL] !== 'cluster-local'
```

Comment:
This works now fine also with private routes, you can drop this check here, right?

Reply:
Needs some discussions on this :)